pyproject.toml

```diff
@@ -48,6 +48,7 @@ test = [
     "pytest",
     "pytest-asyncio>=0.23",
     "pytest-mock",
+    "numpy",
```

Comment:
This should be reverted.

Reply:
Damn, thought I got them all. Thanks for flagging.